native/acter/Cargo.toml

```diff
@@ -64,6 +64,7 @@ features = [
     "anyhow",
     "automatic-room-key-forwarding",
     "bundled-sqlite",
+    "experimental-sliding-sync",
```

Comment:
how come we need that now?

Reply:
I thought `experimental-sliding-sync` is needed for `matrix-sdk-ui`.

Reply:
Is it? Isn't it enough if we just included `matrix-sdk-ui`?

Reply:
Fixed by https://github.com/acterglobal/a3/pull/926/commits/c678baad46ea6d08d05ffa6dff9d6be04300bb63